index.html

```diff
@@ -11,7 +11,21 @@
     />
   </head>
   <body>
-    <header></header>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="#">Hébergements</a></li>
+          <li><a href="#">Activités</a></li>
+        </ul>
+      </nav>
+      <div id="logo_titre">
+        <div id="logo">
+          <img src="assets/img/booki.png" alt="logo Booki" />
+        </div>
+        <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
+        <p>En plein centre ville ou en pleine nature</p>
```

Comment:
Not in header, in search bar pr, rmeove it

Reply:
lines 25 and 26 removed